docs/nodle-wallets/nodle-cash/Nodle_Participation.mdx

```diff
@@ -0,0 +1,47 @@
+
+5. **Building on the Nodle Network:** If you're a developer, you can build IoT applications on top of the Nodle network. By creating useful applications, you contribute to the Nodle ecosystem and promote its use.
+
+Remember that the effectiveness of promotion often lies in spreading awareness and demonstrating value. By actively participating in the Nodle ecosystem and sharing your positive experiences with others, you can contribute significantly to its growth and popularity.
+
+### How can you promote Nodle through writing and vBlogs?
+
+Promoting Nodle through writing and vBlogs can be a very effective way to increase awareness about the network and its benefits. Here are several strategies you can use:
+
+1. **Blog Posts:** Start by writing blog posts about Nodle. These can be introductions to Nodle, tutorials on how to use the Nodle Cash app, or discussions about the benefits of the Nodle network. You can publish these posts on your own website, or submit them to platforms like Medium or Steemit that have large readerships interested in tech and crypto.
+
+2. **Guest Posting:** Reach out to popular tech or cryptocurrency blogs and offer to write a guest post about Nodle. This can help you reach a larger audience.
+
+3. **Social Media Sharing:** Share your posts on social media platforms like Twitter, Facebook, Reddit, or LinkedIn. Tag Nodle's official accounts in your posts to potentially get retweeted or shared by them.
+
+4. **Create vBlogs:** Make video blogs (vBlogs) about Nodle and post them on platforms like YouTube. These could be walkthroughs of the app, discussions about the technology behind Nodle, or interviews with team members. Visual content often gets more engagement, and platforms like YouTube have massive audiences.
+
+5. **Podcasts or Webinars:** Consider starting a podcast or a webinar series where you can discuss topics related to Nodle. Invite experts or Nodle team members for interviews.
+
+6. **Newsletters:** If you have a newsletter, include updates about Nodle in your mailings.
+
+7. **Participate in Discussions:** Join discussions on forums like Reddit or in comments sections of other articles and videos about Nodle. Leave thoughtful comments and link to your own content when relevant.
+
+8. **Tutorial and Explainer Articles/Videos:** Provide a step-by-step guide on how to use Nodle's apps or services, or how to stake Nodle Cash. Tutorials are very popular and can rank highly in search engine results.
+
+9. **Case Studies or Success Stories:** Share your personal experience with the Nodle network. Discuss how you have benefited from it and why you recommend it to others.
+
+In all your content, remember to disclose that you're promoting Nodle to comply with ethical guidelines. Use clear, engaging language, and explain any technical terms for readers or viewers who may be new to blockchain technology.
```

Comment:
I feel like this should be moved to root level instead of being inside the Nodle App folder

Reply:
Moved to root of docs and put in new position